cirq/contrib/quimb/Contract-a-Grid-Circuit.ipynb

```diff
@@ -0,0 +1,405 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Contract a Grid Circuit\n",
+    "Shallow circuits on a planar grid with low-weight observables permit easy contraction."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import networkx as nx\n",
+    "\n",
+    "import cirq\n",
+    "import quimb\n",
+    "import quimb.tensor as qtn\n",
+    "from cirq.contrib.svg import SVGCircuit\n",
+    "\n",
+    "import cirq.contrib.quimb as ccq"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "from matplotlib import pyplot as plt\n",
+    "\n",
+    "import seaborn as sns\n",
```

Comment:
seaborn is missing from the contrib requirements - maybe worth adding it or at least mentioning it?